src/PerformanceTests/Tests/Categories/ConcurrencyFixture.cs

```diff
@@ -6,7 +6,7 @@ namespace Categories
     using Tests.Permutations;
     using Variables;
 
-    [TestFixture(Description = "Concurrency", Category = "Performance"), Explicit]
+    [TestFixture(Description = "Concurrency", Category = "Performance")]
```

Comment:
Potentially we can run this... but these are ALOT of permutations but very valuable non the less. Only thing is that it takes really long before the results are in.

Reply:
I'm happy to selectively exclude some tests (e.g. this one). So far I'm not sure how we could manually run those tests on linux though as the filter API on nunit is not documented.